earthaccess/results.py

```diff
@@ -290,17 +289,20 @@ def data_links(
         """Returns the data links form a granule
 
         Parameters:
-            access: direct or external, direct means in-region access for cloud hosted collections.
-            in_region: if we are running in us-west-2, meant for the store class, default is False
+            access: direct or external.
+              direct means in-region access for cloud-hosted collections.
+            in_region: True if we are running in us-west-2.
+              It is meant for the store class.
```

Comment:
I believe we need two more spaces on these continuation lines.

Reply:
I was following the example from the [Google style guide (section 3.8.3)]. But if this causes problems with the documentation generation, we should certainly add the additional space.

Reply:
Ah. Maybe two spaces does work. Haven't actually tested it. I still prefer to go to 4 for consistency with the rest of the file, or to update the rest of the file to have 2-space indents continuation lines. Not a strong preference though, if it's working!

Reply:
Let's just stick with four spaces, so we don't have to remember "when 2?",  "when 4?", etc.